.github/workflows/docker_tag.yaml

```diff
@@ -42,6 +42,29 @@ jobs:
           file: Dockerfile.doctools
           tags: ${{ steps.meta.outputs.tags }}
 
+  push_releasetools_to_registry:
+    name: Push Docker image to Docker Hub
+    runs-on: ubuntu-latest
```

Comment:
Lepiej dawać precyzyjne wersje, latest odpowiada obecnie `ubuntu-20.04`

Reply:
Can dependabot keep that up-to-date?

Reply:
I'm not sure, but it's fine running outdated versions as long as they're not deprecated - 20.04's EOL is in 2030.